contracts/ICO_controller.sol

```diff
@@ -154,14 +182,16 @@ contract ICO_controller is Ownable {
         crowdsale.burnRemainingTokens();
         totalSold = privateOffer.getWeiRaised().add(preSale.getWeiRaised().add(crowdsale.getWeiRaised()));
         if (totalSold >= SOFTCUP) {
-            token.transfer(incentiveProgram, INCENTIVE_PROGRAM_SUPPORT);
             // sends token for support program
-            token.burn(MAX_DEV_REWARD.sub(totalDevReward));
+            token.transfer(incentiveProgram, INCENTIVE_PROGRAM_SUPPORT);
             // burn some unspent reward tokens
-            owner.transfer(this.balance.div(2));
+            token.burn(MAX_DEV_REWARD.sub(totalDevReward));
+            // burn after airdrop left tokens
+            token.burn(AIRDROP_SUPPLY.sub(AIRDROP_SUPPLY.div(totalAirdropAdrresses).mul(totalAirdropAdrresses)));
             // send 50% of ico eth to contract onwer
-            address(holder).transfer(this.balance);
+            owner.transfer(this.balance.div(2));
             // send other 50% to multisig holder address
+            address(holder).transfer(this.balance);
```

Comment:
Will these two transfers and the requests of balances exactly occur sequentially? May not turn out so getting the balance in the second transfer will happen before the first transfer is complete? Otherwise, the second transfer will not be able to occur due to lack of ethers and ethers will hang on the controller without the ability to take it from there.